doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -718,6 +718,24 @@ the toplevel, and using them in source files is discouraged.
    the :ref:`command line option <command-line-options>` :n:`-I @string` or by the command :cmd:`Add
    ML Path` `@string` (cf. :cmd:`Declare ML Module`).
 
+.. _extra_dependencies:
+
+Extra Dependencies
+------------------
+
+Dependencies on external files, i.e. non ``.v`` files, can be declared as
+follows:
+
+.. cmd:: From @dirpath Extra Dependency @string {? as @ident }
+
+Coq checks that a file name :n:`@string` exists in one, and only one, directory
+associated to :n:`@dirpath`. If :n:`@ident` is provided then Coq gives
+access to the full path of the external file to OCaml plugins via the
+``ComExtraDeps.query_extra_dep`` API.
```

Comment:
```suggestion
Coq checks that a file name :n:`@string` exists in one, and only one, directory
associated with :n:`@dirpath`. If :n:`@ident` is provided, that value can be passed
to the function``ComExtraDeps.query_extra_dep`` to get [EXACTLY WHAT?].
```

The first sentence seems odd--why would you want that behavior?  If I have a string "xyz.o" and there are files "a/xyz.o" and "b/xyz.o", you get an error?
Can the string specify subdirectories, e.g. a string "a/xyz.o"?  If so, they're always separated by "/", even on Windows?